src/tests/Directory.Build.targets

```diff
@@ -658,15 +658,13 @@
     <BuildNativeAotFrameworkObjects Condition="'$(IlcMultiModule)' == 'true'">true</BuildNativeAotFrameworkObjects>
     <DisableFrameworkLibGeneration Condition="'$(BuildNativeAotFrameworkObjects)' == 'true'">true</DisableFrameworkLibGeneration>
     <FrameworkLibPath Condition="'$(BuildNativeAotFrameworkObjects)' == 'true'">$(IlcSdkPath)</FrameworkLibPath>
-
-    <!-- Forced by ILLink targets; we should fix the SDK -->
-    <SelfContained>true</SelfContained>
-    <HasRuntimeOutput>false</HasRuntimeOutput>
-    <_UsingDefaultForHasRuntimeOutput>false</_UsingDefaultForHasRuntimeOutput>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TestBuildMode)' == 'nativeaot'">
     <IlcReference Include="$(TargetingPackPath)/*.dll" />
+
+    <!-- xunit calls MakeGenericType to check if something is IEquatable -->
```

Comment:
The src/tests tree now got also infected with xunit asserts that MakeGenericType a bunch of garbage interfaces only to find the `assert.equal'd` thing doesn't implement such interface. We have to do the same hack in libs tests.